app/models/policies/international_relocation_payments/eligibility_admin_answers_presenter.rb

```diff
@@ -0,0 +1,28 @@
+module Policies
+  module InternationalRelocationPayments
+    class EligibilityAdminAnswersPresenter
+      include Admin::PresenterMethods
+
+      attr_reader :eligibility
+
+      def initialize(eligibility)
+        @eligibility = eligibility
+      end
+
+      def answers
+        [].tap do |a|
+          a << current_school
+        end
+      end
+
+      private
+
+      def current_school
+        [
+          translate("admin.current_school"),
+          eligibility.current_school.present? ? display_school(eligibility.current_school) : "No"
+        ]
```

Comment:
Again not a blocker but we can now assume a current_school is always set on the eligiblity